### DISCUSSION TITLE
Улучшение: эффекты у магических предметов

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2_classic#111] [Ideas]
можно в инстанции предметов добавить эффекты как в НВ

![20230227173602_1](https://user-images.githubusercontent.com/49514764/221630784-0aff7a46-e344-461e-aaf4-88ff2dcdb2aa.jpg)

**Comment by Kor-Angar** [comment]
если не хочешь в сборку можно сделать отдельно в виде мини мода.

[IT_Runen.zip](https://github.com/dosinabox/g2_classic/files/10842153/IT_Runen.zip)

**Comment by dosinabox** [comment]
думаю, не надо
